src/core/renderController.ts

```diff
@@ -93,9 +91,6 @@ export module Core {
         // Layout
         _componentsNeedingComputeLayout.values().forEach((component: Component) => component.computeLayout());
 
-        // Top level render; Containers will put their children in the toRender queue
-        _componentsNeedingRender.values().forEach((component: Component) => component.render());
```

Comment:
Explained already:

Calling `render()` only enqueues the `Component` for rendering, so it can be done synchronously. Calling `render()` on a `ComponentContainer` immediately enqueues all its children, so looping over `_componentsNeedingRender` and forcing them to re-enqueue does nothing.